src/server/webkit/wkPage.ts

```diff
@@ -1083,14 +1082,6 @@ export class WKPage implements PageDelegate {
     this._page._frameManager.requestFailed(request.request, event.errorText.includes('cancelled'));
   }
 
-  _onDataReceived(event: Protocol.Network.dataReceivedPayload) {
-    const request = this._requestIdToRequest.get(event.requestId);
-    if (!request)
-      return;
-    request.request.responseSize.bodySize += event.dataLength || (event.encodedDataLength === -1 ? 0 : event.encodedDataLength);
```

Comment:
This suggests we don't get any information for WebKit at all?

Reply:
Yeah WK onDataReceived is half defect since it only reflects the decodedBodySize.

Reply:
Yeah, but let's at least use metrics for Mac to work?
```
  requestHeaderBytesSent: 368,
  requestBodyBytesSent: 0,
  responseHeaderBytesReceived: 206,
  responseBodyBytesReceived: 64,
  responseBodyDecodedSize: 5100,
  securityConnection: { protocol: 'SSL 2.0', cipher: 'TLS_NULL_WITH_NULL_NULL' }
```

Reply:
follow up PR to use sizes from the browsers in CR and WK.